third_party/move/move-compiler-v2/tests/ability-check/v1-typing/constraints_not_satisfied_lvalues_pack_type_args.exp

```diff
@@ -10,3 +10,12 @@ error: type `R` is missing required ability `copy`
   │                    ^
   │
   = required by instantiating type parameter `phantom T:copy` of struct `CupC`
+
+warning: Unnecessary trailing semicolon
```

Comment:
V1 does not generate a different kind of error: https://github.com/aptos-labs/aptos-core/blob/main/third_party/move/move-compiler/tests/move_check/typing/constraints_not_satisfied_lvalues_pack_type_args.exp
Is this a breaking change?

Reply:
v1 has 4 errors:
- line 8: ability constraint: `The type '0x8675309::M::R' does not have the ability 'copy'`
- line 8: dead or unreachable code: `abort 0` is divergent, code following this will be dead.
- line 9: ability constraint: `The type '0x8675309::M::R' does not have the ability 'copy'`
- line 9: dead or unreachable code: divergent expression, code following this will be dead

v2 previously has 1 error:
- line 8:  type `R` is missing required ability `copy`

We don't yet have a "dead code" warning.  Maybe I'll have to integrate the "Invalid use of a divergent expression" warning into the current PR.  Hold on.